packages/babel-plugin-transform-react-jsx-development/test/fixtures/linux/classic-runtime-with-source-self-plugins/output.js

```diff
@@ -0,0 +1,38 @@
+var _jsxFileName = "<CWD>/packages/babel-plugin-transform-react-jsx-development/test/fixtures/linux/classic-runtime-with-source-self-plugins/input.js",
+    _jsxFileName2 = "<CWD>/packages/babel-plugin-transform-react-jsx-development/test/fixtures/linux/classic-runtime-with-source-self-plugins/input.js";
```

Comment:
The first one is injected because of `transform-react-jsx-source` plugin, and it's not used because we filter out the injected source and self and insert our own. We could throw if someone uses the new transform and source or self, but that seems a bit aggressive. What doo you think?